packages/smooth_app/lib/pages/onboarding/onboarding_flow_navigator.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:smooth_app/data_models/user_preferences.dart';
+import 'package:smooth_app/pages/onboarding/scan_example.dart';
+import 'package:smooth_app/pages/onboarding/welcome_page.dart';
+import 'package:smooth_app/pages/page_manager.dart';
+
+enum OnboardingPage {
+  NOT_STARTED,
+  WELCOME,
+  SCAN_EXAMPLE,
+  ONBOARDING_COMPLETE,
+}
+
+/// Decide which page to take the user to.
+class OnboardingFlowNavigator {
+  OnboardingFlowNavigator(this._userPreferences);
+
+  final Future<UserPreferences> _userPreferences;
+
+  Widget getNextPageWidget(BuildContext context, OnboardingPage currentPage) {
+    switch (currentPage) {
+      case OnboardingPage.NOT_STARTED:
+        // First screen, doesn't have a back navigation button.
+        return const WelcomePage();
+        break;
```

Comment:
The `break` after a `return` is pointless; please remove it.

Reply:
already fixed, you're review was too fast :D